src/Stratis.Bitcoin/P2P/Peer/NetworkPeerServer.cs

```diff
@@ -234,7 +238,7 @@ private NetworkPeerConnectionParameters CreateNetworkPeerConnectionParameters()
         /// </summary>
         /// <remarks>Nodes catching up to the longest chain are vulnerable to malicious peers serving up a fake longer chain.</remarks>
         /// <returns><c>true</c> when the client is closed.</returns>
-        private bool CloseClientNotWhiteListedDuringIBD(TcpClient tcpClient)
+        private bool AllowClientConnection(TcpClient tcpClient)
```

Comment:
This method should not close the connection, I mentioned it would be nice if it included the condition on line 171